plugins/jobs/server/models/job.py

```diff
@@ -79,13 +97,11 @@ def listAll(self, limit=0, offset=0, sort=None, currentUser=None):
         :param offset: The page offset
         :param sort: The sort field.
         :param currentUser: User for access filtering.
+        .. deprecated :: 2.3.0
+           Use :func:`list` instead
```

Comment:
is there a way to specify that this is the `list` method of the `Job` class, instead of say, Python's `list` function?  

It is probably fine as is, but if it is easy to add in that additional info, please do.